docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+* Make it easy to imitate plugins as they build their own.
+    * The profile cards and pages should easily link to the Github repo.
+* Recommend plugins based on their needs.
+    * You could fill out a profile and login and get suggestions based on the kind of site you’re building, your level of experience, etc.
+    * There could be permanent categories like “for beginners” or other categories.
+    * There can also be changing categories like “editor’s picks,” “trending,” “most popular,” etc.
+* Show (or at least explain) what plugins do (preview) before installation.
+    * Make sure README files are filled out with instructions and a list of sites that use the plugin.
+    * Possible showcase every week?
+* Make it easy to share plugins they build and/or plugins they like.
+    * A huge thank you to @jlengstorf and all involved in building the [Plugin Authoring page in the docs](https://www.gatsbyjs.org/docs/plugin-authoring/#what-files-does-gatsby-look-for-in-a-plugin)
+    * Share buttons from the profile card and/or profile page.
+
+
+## Learning from other plugin libraries
+
+We also analyzed over 10 admirable and/or popular plugin libraries to draw from their strengths and learn from their weaknesses. Examples in no particular order include [JS.coach](https://js.coach/), [VIM Awesome](https://vimawesome.com/), [Best of JS](https://bestof.js.org/), [Sketch extension library](https://sketchapp.com/extensions/), [Chrome plugin library](https://chrome.google.com/webstore/detail/plugins/mmcblfncjaclajmegihojiekebofjcen?hl=en), [Microsoft Visual Studio Code Extensions](https://marketplace.visualstudio.com/VSCode), [Wordpress Plugins](https://wordpress.org/plugins/), [Npms.io](https://npms.io/), [Yarnpkg.com](https://yarnpkg.com/en/packages), [Apple App store](https://www.apple.com/ios/app-store/), which just got redesigned, so I’m interested to learn from their changes, [Google Play store](https://play.google.com/store/apps/top).
+
+
+## Card sorting
+
+We gathered, and are continuing to gather, feedback from Github issues, the Product Hive slack community, and face-to-face usability testing. Also, as part of our plugin library UX research, I wanted to know what keywords people might use to search for Gatsby plugins in order to create filters, categories, and tags that make sense. 
+
+I used card sort software to generate dendrograms, other kinds of -grams, and matrices that show the way 20 different Gatsby users typically organize plugins in their minds (big thank you to all who participated here!). Creating filters, categories, and tags are possible future enhancements based off of these results: 
+* [Card Sort Results 1](https://www.optimalworkshop.com/optimalsort/x87kpp82/0mv7111b/shared-results/f24tt2xcxj7di7303t46hec077unam75)
+* [Card Sort Results 2](https://www.optimalworkshop.com/optimalsort/x87kpp82/bt0c01w0/shared-results/1yo4d868hponypjyk1we3xz53sag7uc7)
```

Comment:
Unclear what these links are trying to demonstrate -- this seems to be a statistical analysis which which I'm unfamiliar and the results of which aren't clear to me. If the analysis was useful and suggested a scheme of 5-10 filters / categories / tags I would just list them here. If not, this is confusing.

Reply:
Ok. Yeah, the results are pretty obscure. Should we just leave this out?